Added "-fPIC" to CFLAGS in order to allow iscsiuio to build successfully

--- a/iscsiuio/configure.ac
+++ b/iscsiuio/configure.ac
@@ -52,7 +52,7 @@
 # libtool stuff
 AC_PROG_LIBTOOL
 
-CFLAGS="-O2 -Wall"
+CFLAGS="-O2 -Wall -fPIC"
 ## check for --enable-debug first before checking CFLAGS before
 ## so that we don't mix -O and -g
 AC_ARG_ENABLE(debug,
